hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -53,6 +53,8 @@ public final class OzoneConsts {
   public static final String CLUSTER_ID = "clusterID";
   public static final String LAYOUTVERSION = "layOutVersion";
   public static final String CTIME = "ctime";
+
+  public static final String PATH_DELIMITER = "/";
```

Comment:
Please don't create another constant. We already have [OZONE_URI_DELIMITER](https://github.com/apache/ozone/blob/master/hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java#L102).

Reply:
done